pkg/pool-stable-phantom/contracts/StablePhantomPool.sol

```diff
@@ -232,15 +232,28 @@ contract StablePhantomPool is IRateProvider, BaseGeneralPool, ProtocolFeeCache {
         // reference them by token index in the full base tokens plus BPT set (i.e. the tokens the Pool registers). Due
         // to immutable variables requiring an explicit assignment instead of defaulting to an empty value, it is
         // simpler to create a new memory array with the values we want to assign to the immutable state variables.
+        IRateProvider[] memory rateProviders = new IRateProvider[](params.tokens.length + 1);
+        // Do the same with exemptFromYieldProtocolFeeFlags
+        bool[] memory exemptFromYieldFlags = new bool[](rateProviders.length);
+
         IRateProvider[] memory tokensAndBPTRateProviders = new IRateProvider[](params.tokens.length + 1);
         for (uint256 i = 0; i < tokensAndBPTRateProviders.length; ++i) {
             if (i < bptIndex) {
-                tokensAndBPTRateProviders[i] = params.rateProviders[i];
+                rateProviders[i] = params.rateProviders[i];
+                exemptFromYieldFlags[i] = params.exemptFromYieldProtocolFeeFlags[i];
             } else if (i == bptIndex) {
-                tokensAndBPTRateProviders[i] = IRateProvider(0);
+                rateProviders[i] = IRateProvider(0);
```

Comment:
```suggestion
                rateProviders[i] = IRateProvider(0);
                exemptFromYieldFlags[i] = false;
```

This would make it consistent with the rate providers.

Reply:
It's already like this in 1496.